ui/app/sample-data/demoDashboard.ts

```diff
@@ -159,47 +159,19 @@ const demoDashboard: DashboardResource = {
                       kind: 'PrometheusTimeSeriesQuery',
                       spec: {
                         query:
-                          'node_memory_MemTotal_bytes{job="node",instance=~"$instance"} - node_memory_MemFree_bytes{job="node",instance=~"$instance"} - node_memory_Buffers_bytes{job="node",instance=~"$instance"} - node_memory_Cached_bytes{job="node",instance=~"$instance"}',
-                      },
-                    },
-                  },
-                },
-                {
-                  kind: 'TimeSeriesQuery',
-                  spec: {
-                    plugin: {
-                      kind: 'PrometheusTimeSeriesQuery',
-                      spec: {
-                        query: 'node_memory_Buffers_bytes{job="node",instance=~"$instance"}',
-                      },
-                    },
-                  },
-                },
-                {
-                  kind: 'TimeSeriesQuery',
-                  spec: {
-                    plugin: {
-                      kind: 'PrometheusTimeSeriesQuery',
-                      spec: {
-                        query: 'node_memory_Cached_bytes{job="node",instance=~"$instance"}',
-                      },
-                    },
-                  },
-                },
-                {
-                  kind: 'TimeSeriesQuery',
-                  spec: {
-                    plugin: {
-                      kind: 'PrometheusTimeSeriesQuery',
-                      spec: {
-                        query: 'node_memory_MemFree_bytes{job="node",instance=~"$instance"}',
+                          '1 - node_filesystem_free_bytes{job="node",instance=~"$instance",fstype!="rootfs",mountpoint!~"/(run|var).*",mountpoint!=""} / node_filesystem_size_bytes{job="node",instance=~"$instance"}',
+                        legendFormat: '{{instance}} {{job}}', // figure this syntax out with Shan / Luke
```

Comment:
To support showing/hiding queries on charts*, we'll need some `is_hidden` property too.

We could do:
```
spec: {
  display: {
    legend_format: ...,
    is_hidden: ...,
  },
  query: ...,
}
```
Or:
```
spec: {
  legend_format: ...,
  is_hidden: ...,
  query: ...,
}
```

* I am 95% sure that is the intention of the toggle in this design, but I'll talk to Mohit to confirm.
<img width="874" alt="Screen Shot 2022-10-04 at 12 44 40 PM" src="https://user-images.githubusercontent.com/2584129/193911443-493bacdd-af6e-4e09-b2c8-ba10fa50f737.png">

Reply:
Ohh gotcha, yeah I was going to say we'd just leave it out of the queries array when we don't want it to show, but forgot about this disabled state

Reply:
I think I slightly prefer the second option where it isn't inside display, but curious what everyone else thinks

Reply:
I would go for the second option too. `display` seems wrong regarding the effect of `hidden`. 

By the way by `is_hidden`, do we actually mean "deactivate the query` so it's not ran? Or do we run the query and then we don't just display the result. 

and `display` could conflict with the other `display` coming from the chart itself.